includes/class-wc-payments-fraud-service.php

```diff
@@ -188,4 +207,24 @@ private function get_cookie_session_id() {
 		$cookie_customer_id = $cookie[0];
 		return $wpcom_blog_id . '_' . $cookie_customer_id;
 	}
+
+	/**
+	 * If a "forterToken" cookie is present, send it to the WCPay server so the
+	 * current browsing session can be linked to the account. It will only be sent once.
+	 */
+	public function send_forter_cookie_token() {
+		if ( ! $this->account->is_stripe_connected() || ! isset( $_COOKIE['forterToken'] ) ) {
```

Comment:
You're right on both counts. Completely an edge case, but it's trivial to prevent :)

Done in https://github.com/Automattic/woocommerce-payments/pull/1349/commits/10780f2f3758e1a0c751c9a97b5be7f24bb7ec3e